pcs/lib/env.py

```diff
@@ -176,20 +197,6 @@ def cib(self):
     def get_cluster_state(self):
         return get_cluster_status_dom(self.cmd_runner())
 
-    def get_wait_timeout(self, wait):
-        if wait is False:
-            return False
-
-        if wait not in self.__timeout_cache:
-            if not self.is_cib_live:
-                raise LibraryError(
-                    ReportItem.error(
-                        reports.messages.WaitForIdleNotLiveCluster()
-                    )
-                )
-            self.__timeout_cache[wait] = get_valid_timeout_seconds(wait)
-        return self.__timeout_cache[wait]
```

Comment:
`__timeout_cache` doesn't seem to be used anywhere, so it can be dropped from `__init__` as well